### ISSUE TITLE
Add linkedin , instagram , twitter links at the bottom

### ISSUE THREAD

**Issue by vishapraj** [Sriparno08/Openpedia#215]
## Category

- [ ] Documentation
- [ ] Resource Addition
- [ ] Codebase
- [ ] User Interface
- [ ] Feature Request

**Comment by vishapraj** [comment]
Pls assign me this issue under JWOC.

**Comment by Sriparno08** [comment]
@vishapraj You're assigned.

**Comment by vishapraj** [comment]
pls provide me your insta id link.

**Comment by Sriparno08** [comment]
I don't have one, @vishapraj. That's why, an issue is already opened to remove the 'Instagram' icon from the landing page footer.

**Comment by vishapraj** [comment]
ok , I will remove instagram icon

**Comment by Sriparno08** [comment]
Hey @vishapraj, don't remove it as part of this issue since we already have another issue opened that addresses this topic.

You can just add the links and make a PR.

**Comment by Sriparno08** [comment]
You can only be assigned to one issue at a time. Once your PR for this issue is merged, you will be considered for other issues.

**Comment by vishapraj** [comment]
ok, pls merge my PR

**Comment by Sriparno08** [comment]
@vishapraj You haven't made a PR for this issue yet.

**Comment by Divyanshgarg-04** [comment]
@Sriparno08 I would love to work on this issue under JWOC. Please assign this issue to me.

**Comment by vermaanurag1532** [comment]
please Assign it to me
